### ISSUE TITLE
Can't send HTML or Markdown statuses

### ISSUE THREAD

**Issue by CPlusPatch** [h3poteto/megalodon#1596]
Every time I use `postStatus`, there is no way for me to set the content-type as "plain", "markdown", "html" or anything else.

I have analyzed the HTTP requests performed by Megalodon and they are missing the content_type POST parameter, which is `text/plain` on normal messages.

**Comment by h3poteto** [comment]
Is it allowed?
Can you send markdown or HTML on WebUI?

**Comment by CPlusPatch** [comment]
Absolutely, I use Akkoma (Pleroma fork, same API) and I can send Markdown or HTML or MFM messages from within the default web interface

**Comment by h3poteto** [comment]
What happens when you send Markdown or HTML in the message? It is rendered as HTML in the timeline?
When I post markdown text in Mastodon, it is not rendered as HTML, it is rendered as plain text: https://mastodon-japan.net/@h3poteto/109930646032098298

**Comment by CPlusPatch** [comment]
I use Akkoma (a Pleroma fork) and Calckey (Misskey fork) which render HTML text as HTML

Besides, Mastodon has just merged support for HTML text rendering as well (https://github.com/mastodon/mastodon/pull/23913)

**Comment by SeolHa314** [comment]
Glitch-soc and some Pleroma forks supports markdown formatting by setting `content-type` param, is there a plan to add this feature in megalodon?
